imagebuildah/build.go

```diff
@@ -1225,3 +1236,14 @@ func BuildDockerfiles(ctx context.Context, store storage.Store, options BuildOpt
 	stages := imagebuilder.NewStages(mainNode, b)
 	return exec.Build(ctx, stages)
 }
+
+// deleteSuccessfulIntermediateCtrs goes through the container IDs in b.containerIDs
+// and deletes the containers associated with that ID.
+func (b *Executor) deleteSuccessfulIntermediateCtrs() error {
+	for _, ctr := range b.containerIDs {
```

Comment:
Add lasterror here, so that it can report the error.  Otherwise this always returns nil.

Reply:
But this function doesn't need to return anything since we are just logging the errors right? The error is returned if it occurs during the execute or prepare step above.

Reply:
nvm, I got what you meant. Fixed.